### PR TITLE
When storing events, fallback to an existing events idea of the batches first.

### PR DESCRIPTION
**Pull request by hodgestar**
Currently we always look up the outbound message.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: vumi/components/message_store.py [hunk 1]
```diff
@@ -424,26 +424,33 @@ def get_outbound_message(self, msg_id):
...
         event_id = event['event_id']
         msg_id = event['user_message_id']
         event_record = yield self.events.load(event_id)
         if event_record is None:
             event_record = self.events(event_id, event=event, message=msg_id)
+            if batch_ids is None:
+                # If we aren't given batch_ids, get them from the outbound
+                # message.
+                batch_ids = yield self._get_batches_from_outbound(msg_id)
+                print batch_ids
```
Comment (smn) [inline vumi/components/message_store.py]: stray print

**Comment by smn** [comment]
:+1: from me but I'm a bit behind on these internals.

**Comment by jerith** [comment]
:+1: based on local test runs. Travis builds are stalled and it'll be a while before we see any happen. :-/

**Pull request merged**